src/renderer/features/proxies/lib/__tests__/proxies-utils.test.ts

```diff
@@ -22,4 +22,37 @@ describe('features/proxies/lib/proxies-utils', () => {
 
     expect(result).toEqual(false);
   });
+
+  test('should check if proxy for account is available (true for chain account)', () => {
```

Comment:
I don't like that first half of test name is kind of meaningless and only second part is actually states what test does
Try this one: "should be true if proxy is available for chain account" and so on 😉

Reply:
I will rename it, but I think we should discuss naming convention for tests